src/static/riot/datasets/management.tag

```diff
@@ -152,19 +152,19 @@
             </div>
 
             <form class="ui form coda-animated {error: errors}" ref="form">
```

Comment:
you could put `selenium='dataset-form'` then do lookups like... `form[selenium="dataset-form"] input[name="name"]` to lookup the name input. This makes it a bit easier to maintain, that way if we add 10 new fields we don't need 10 new `selenium='scoring-new-field-name'`